orchestration/plugins/lookup_plugins/vault.py

```diff
@@ -0,0 +1,31 @@
+from ansible.plugins.lookup import LookupBase
+from ansible.errors import AnsibleError
+
+from hvac import Client
+import json
+import os
+import re
+
+# Modeled after https://github.com/jhaals/ansible-vault/blob/master/vault.py
+
+
+class LookupModule(LookupBase):
```

Comment:
curious what this `LookupModule` is for with ansible - it seems some of the logic here is duplicated with what's in `get_vault_credentials_from_path` from `utils`